### ISSUE TITLE
ClientContract.getClients does not seems to work correctly

### ISSUE THREAD

**Issue by rolandorh** [verified-network/verified-sdk#15]
**ClientContract.getClients does not seems to work correctly**

- I have two wallets set up in Ropsten, let's call them Issuer and Investor.
- I already set Issuer as manager of Investor.
- When I call getClients on Issuer ClientContract, it does not return Investor address as expected.

Below is the code to repoduce it:

```javascript
const {KYCContract, ClientContract, VerifiedWallet, Provider} = require('@verified-network/verified-sdk');

(async function main() {
    const provider = Provider.infuraProvider('ropsten', 'd9e961d8faa94e8780d8bee68eb737f2');

    // 0x7079C0a4CFECf0B575eaF7C103AfA197e847B217 : 0x749bae20fb192f0be8f3f85f56d8f9cfd1bafe1a21f5dc02c84a8f59c551b4bc
    const investorMnemonics = 'empty foil embrace forest duty witness shaft favorite ostrich poverty online frozen';

    // 0xFe59f26Ea8DB34f2ddB82594007fa8B2098180a2 : 0xd00f8ef75b7ea5b3a6810fdeabd3f9d809563d438213110562511490f0e96362
    const issuerMnemonics = 'shoe steak sample jeans motion air hard input thrive spy trumpet snack';

    // This two account are being used for test, where Issuer is already manager for Investor
    let investorWallet = VerifiedWallet.importWallet(investorMnemonics).setProvider(provider);
    let issuerWallet = VerifiedWallet.importWallet(issuerMnemonics).setProvider(provider);

    const clientContract = new ClientContract(issuerWallet);

    const investorManager = (await clientContract.getManager(investorWallet.address)).response.result[0];

    // Issuer is already manager for investor
    console.log('Investor manager is: ', investorManager);
    console.assert(investorManager === issuerWallet.address, 'Issuer is manager of investor');

    // Expected investor address to be in one of this responses, but getting two empty arrays instead
    const clientsWithKyc = (await clientContract.getClients(issuerWallet.address, true)).response.result;
    const clientsWithoutKyc = (await clientContract.getClients(issuerWallet.address, false)).response.result;
    // const clientsWithKyc = (await clientContract.getClients(issuerWallet.address, true)).response.result[0]; // Or maybe in this way
    // const clientsWithoutKyc = (await clientContract.getClients(issuerWallet.address, false)).response.result[0]; // Or maybe in this way

    console.log('Clients with KYC: ', clientsWithKyc);
    console.log('Clients without KYC: ', clientsWithoutKyc);

    console.assert(clientsWithoutKyc.length > 0 || clientsWithKyc.length > 0, 'At least one of them should not be empty');
})();
```

**Comment by kallolborah** [comment]
@rolandorh can you also put the request/response that you see in the browser debug console so that we can check if the call is reaching the SDK function or not, or whether SDK is throwing exception, or whether there is an issue with the contract ?

**Comment by kallolborah** [comment]
@rolandorh fixed this bug, it was in the smart contract. Bug fix to be available in the next release of SDK v0.0.9
